src/main/java/com/wynntils/modules/core/commands/CommandServer.java

```diff
@@ -65,6 +73,78 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
         }
     }
 
+    private void nextSoulPoint(MinecraftServer server, ICommandSender sender, String[] args){
+        Map<String, Integer> nextServers = new HashMap<>();
+
+        for (String availServer : ServerListManager.getAvailableServers().keySet()) {
+            int uptimeMinutes = ServerListManager.getUptimeTotalMinutes(availServer);
+            if ((uptimeMinutes % 20) >= 10) { // >= to 10min because we're looking for time UNTIL 20min, not after
+                nextServers.put(availServer, 20 - (uptimeMinutes % 20));
+            }
+        }
+
+        // Sort servers by time until soul point
+        Map<String, Integer> sortedServers = nextServers.entrySet().stream()
+                .sorted(Map.Entry.comparingByValue())
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue, (e1, e2) -> e1, LinkedHashMap::new));
+
+
+        TextComponentString toSend = new TextComponentString("\n" + "§bApproximate soul point times:" + "\n");
```

Comment:
No need for a newline before the message. Also no need to set it to bold, you do it with a style later on.

```suggestion
        TextComponentString toSend = new TextComponentString("Approximate soul point times:" + "\n");
```